src/quo/components/buttons/button/view.cljs

```diff
@@ -53,81 +53,84 @@
         on-press-out-cb (rn/use-callback
                          (fn []
                            (set-pressed-state nil)
-                           (when on-press-out (on-press-out))))]
+                           (when on-press-out (on-press-out))))
+        button-content
```

Comment:
could you please elaborate on this change

Reply:
ah not necessary anymore, initially i had done this to move `touchable-without-feedback` to `pressable`
But as i can now see that @seanstrom fixed those in his PR : https://github.com/status-im/status-mobile/pull/21874

I will revert this change.